core/src/App.html

```diff
@@ -491,13 +491,32 @@
 
   export const openSearchField = () => {
     isSearchFieldVisible = true;
+    if (document.querySelector('.luigi-search__input')) {
+      document.querySelector('.luigi-search__input').focus();
+    }
   };
```

Comment:
Do you like such approach to avoid duplication of the query selector (simpler unit testing?), which you can reuse in all the functions or is it over-engineered..

```suggestion
const searchInputElem = () => document.querySelector('.luigi-search__input');

export const openSearchField = () => {
    isSearchFieldVisible = true;
    if (searchInputElem()) {
      searchInputElem().focus();
    }
  };
```